docs: create a kubernetes section (#5315)
jina-ai/jina: docs/how-to/kubernetes.md

- [Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service)
- [Digital Ocean](https://www.digitalocean.com/products/kubernetes/)

> And you need to install Linkerd in your K8s cluster.  To use Linkerd, [install the Linkerd CLI](https://linkerd.io/2.11/getting-started/).
> After that, [install its control plane](https://linkerd.io/2.11/getting-started/) in your cluster.
> This automatically sets up and manages the service mesh proxies when you deploy the Flow.

Reviewer: ```suggestion
You need to install Linkerd in your K8s cluster. To use Linkerd, [install the Linkerd CLI](https://linkerd.io/2.11/getting-started/) and [its control plane](https://linkerd.io/2.11/getting-started/) in your cluster.
This automatically sets up and manages the service mesh proxies when you deploy the Flow.
```

## Key takeaways

> To put it briefly, there are just three key steps about deploying a Jina Flow on Kubernetes:

Reviewer: ```suggestion
In short, there are just three key steps to deploy a Jina Flow on Kubernetes:
```
